doc/level1.rst

```diff
@@ -417,6 +417,12 @@ public keys.  It MAY support other OpenPGP key formats found in
 a ``type=1`` Autocrypt header (for example, by passing it agnostically
 to an OpenPGP backend for handling).
 
+We recommend against protecting the private key with a password. All
+encrypted outgoing messages MUST also be signed, which would require the
+user to enter their password for both reading and sending mail. This
+introduces too much friction to become part of a routine daily workflow.
+Protection of the user's keys at rest is achieved more easily and
+securely with full-disk encryption.
```

Comment:
I wouldn't add the second part. While true, it weakens our point for architectures that *don't* have that property. I like the first idea though, how about just adding "and other files"?:

"Protection of the user's keys and other files at rest is achieved more easily and securely with full-disk encryption."